### ISSUE TITLE
Port(UDP, UART) does not open on 1.0.0beta2

### ISSUE THREAD

**Issue by OpusK** [eProsima/micro-RTPS-agent#1]
Hi,

I am testing with 1.0.0beta2 which I recently updated. There is a problem, and when I check it, the Agent can not open the Port(UDP, UART).

For UDP, I checked with wireshark that "destination unreachable" message(ICMP packet) is being returned.
For UART, I created and tested the main code, which does not work if the UART port does not open, but it did not work when running the Agent (version 1.0.0 beta works this way).
Also, if an attempt is made to open the same port with a serial program while the Agent is running, an error should occur. However, the serial program opens the port and communicates!

Do you know anything about this?

**Comment by OpusK** [comment]
I tested with udp port number 2020, 2018.

**Comment by julianbermudez** [comment]
Hi @OpusK, 
I tested the UDP server in Agent with `udp 2018` and it works for me with the current version or release/1.0.0beta2 branch. Could you confirm that it works for you?

**Comment by OpusK** [comment]
Hi, @julianbermudez 

master version is working well.
But, release/1.0.0beta2 branch version is not working.

I have confirmed that the currently open port is open with loopback ip.

> udp        0      0 127.0.0.1:2018          0.0.0.0:*                           452/MicroRTPSAgent

I have been trying to connect to remote ip from now on.
In master version, I did not check which ip port is open because it worked well then. 

Anyway, is it correct to open in loopback ip?
In this condition, it is correct that remote can not reach this port.
